examples/ethereum_json_rpc/src/main.py

```diff
@@ -0,0 +1,84 @@
+from kybra import Async, CanisterResult, ic, init, nat32, query, update
+from kybra.canisters.management import management_canister
+from kybra.canisters.management.http import HttpResponse, HttpTransformArgs
+from typing import TypeAlias, TypedDict
+
+JSON: TypeAlias = str
```

Comment:
Why the explicit `TypeAlias` annotation?

Reply:
I just like being explicit, I can get rid of that